src/Message/Lists/Action.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Netflie\WhatsAppCloudApi\Message\Lists;
```

Comment:
We're using singular naming, please, can you modify the `Lists` folder to `List` and it namespace? Thanks!

Reply:
I wasn't able to rename it from `Lists` to `List`, since `List` is a reserved keyword prior to PHP 8.0. What I did was renaming the folder from `Lists` to `OptionsList`, which is already singular.